invenio_rdm_records/vocabularies/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
this module is not needed anymore, awards are registered with the facet directly in invenio-vocabularies